include/librfn/protothreads.h

```diff
@@ -120,6 +120,7 @@ typedef uint16_t pt_t;
 #define PT_WAIT_UNTIL(c)                                                       \
 	do {                                                                   \
 		*missing_PT_BEGIN = __LINE__;                                  \
+	/* FALLTHROUGH */                                            \
```

Comment:
Good change.

Nit picking but I prefer it spelled as FALLTHRU and aligned like a break not a case (even for funky uses like this one).